react-native-text-input-mask.podspec

```diff
@@ -24,6 +24,5 @@ Pod::Spec.new do |s|
     s.requires_arc  = true
     s.swift_version = "5.0"
     s.dependency 'React-Core'
-    s.dependency 'React-RCTText'
```

Comment:
Interesting that this is no longer required, i had a lot of build issues before, and this dependency seemed to fix it, but will test without it